pkg/plugininventory/sqlite_inventory.go

```diff
@@ -222,7 +222,7 @@ func createPluginWhereClause(filter *PluginInventoryFilter) (string, error) {
 				whereClause = fmt.Sprintf("%s Version=RecommendedVersion AND", whereClause)
 			} else {
 				// We want a specific version of the plugin
-				whereClause = fmt.Sprintf("%s Version='%s' AND", whereClause, filter.Version)
+				whereClause = fmt.Sprintf("%s Version Like '%s%%' AND", whereClause, filter.Version)
```

Comment:
We have to be careful that a version of e.g., `v0.2` doesn’t match e.g.,`v0.28.1` but only matches `v0.2.PATCH`

This type of query probably deserves a comment if you agree.

Reply:
Thanks for catching this. I have updated the query and added a comment explaining this condition.